core/src/main/java/edu/wpi/grip/core/settings/CodeGenerationSettings.java

```diff
@@ -0,0 +1,160 @@
+package edu.wpi.grip.core.settings;
```

Comment:
Or just slap two lombok annotations on it and remove everything except the fields...